packages/design-system/src/components/MultiSelectField/MultiSelect.js

```diff
@@ -396,7 +396,7 @@ class MultiSelect extends React.PureComponent {
       }),
       className
     );
-
+    const headItemsStyles = selectedItems && selectedItems.length > 0 ? `${baseClass}-head__items` : `${baseClass}-head__items-empty`;
```

Comment:
don't you need selectHeader in the MUltiselect as well?

Reply:
for now not, in multiselect we have "select/deselect all" row, adding header could mess up.